### DISCUSSION TITLE
Please support kde file browser or use the rct2 gui one

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [OpenRCT2/OpenRCT2#16803] [Feature requests]
The file browser you use doesn't look native at all and even worse it can't find 99% of my files and folders.
I'm on kde neon 20.04
![image](https://user-images.githubusercontent.com/2453707/158529228-35395429-d593-4d7e-b297-03bb97adc837.png)

**Comment by Shadowblitz16** [comment]
Odd The gog option uses the correct file dialog. reporting this as a bug....

**Comment by Shadowblitz16** [comment]
posted the bug report here https://github.com/OpenRCT2/OpenRCT2/issues/16804

**Comment by J-PIE-314** [comment]
https://github.com/OpenRCT2/OpenRCT2/issues/16804 has been solved, does that mean that the discussion here has been answered as well?

**Reply by Shadowblitz16** [reply]
I haven't played openrct2 in a long time this can be closed and I can just make a new issue if it still exists nest time I play.
